docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -542,6 +547,66 @@ Name                                                  Current Value
 > [!NOTE]
 > OpenTelemetry refers to tags as 'attributes'. These are two different names for the same functionality.
 
+## Using Advice to customize Histogram instruments
+
+Consumers of Histogram instruments can choose different aggregation strategies.
+The default mode when using OpenTelemetry is [Explicit Bucket Histogram
+Aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation).
+
+When using explicit bucket aggregation the measurements recorded through a
+Histogram will be grouped into buckets which track the sum and count of values
+to that bucket.
+
+The default bucket configuration is described in the OpenTelemetry
+Specification: `[ 0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000,
+7500, 10000 ]`.
+
+The default values may not lead to the best granularity for every Histogram. For
```

Comment:
https://opentelemetry.io/blog/2023/why-histograms/
https://opentelemetry.io/blog/2023/histograms-vs-summaries/

are 2 useful links that maybe shared here for users to read more on this area...

Reply:
Added links as a sub-section after the main content